### PR TITLE
CapMan predefined queries

### PR DESCRIPTION
**Pull request by xurui-c**
Adds a predefined query to Snuba Admin's Querylog. This is for the CapMan runbook for when we need to increase a customer's threshold.

### CODE REVIEW

**Review by volokluev — APPROVED** [review]

File: snuba/admin/clickhouse/predefined_querylog_queries.py [hunk 1]
```diff
@@ -371,3 +371,51 @@ class TotalBytesScannedByReferrerAndProject(QuerylogQuery):
...
+        JSON_QUERY(arrayJoin(clickhouse_queries.stats), '$.quota_allowance.summary.rejection_storage_key')
+            AS rejection_storage,
+        JSON_QUERY(arrayJoin(clickhouse_queries.stats), '$.quota_allowance.summary.rejected_by.policy')
+            AS rejection_policy,
+        referrer,
+        projects,
+        COUNT(*) AS count
+        FROM querylog_dist
+        WHERE toDateTime('{{start_timestamp}}', 'Universal') < timestamp
+        AND timestamp < toDateTime('{{end_timestamp}}', 'Universal')
```
Comment (onkar) [inline snuba/admin/clickhouse/predefined_querylog_queries.py:392]: Should this be to include `start_timestamp` matching `timestamp` exactly :
```
WHERE toDateTime('{{start_timestamp}}', 'Universal') <= timestamp
AND timestamp <= toDateTime('{{end_timestamp}}', 'Universal')
```

File: snuba/admin/clickhouse/predefined_querylog_queries.py [hunk 2]
```diff
@@ -371,3 +371,51 @@ class TotalBytesScannedByReferrerAndProject(QuerylogQuery):
...
+        JSON_QUERY(arrayJoin(clickhouse_queries.stats), '$.quota_allowance.summary.throttle_storage_key')
+            AS throttle_storage,
+        JSON_QUERY(arrayJoin(clickhouse_queries.stats), '$.quota_allowance.summary.throttled_by.policy')
+            AS throttle_policy,
+        referrer,
+        projects,
+        COUNT(*) AS count
+        FROM querylog_dist
+        WHERE toDateTime('{{start_timestamp}}', 'Universal') < timestamp
+        AND timestamp < toDateTime('{{end_timestamp}}', 'Universal')
```
Comment (onkar) [inline snuba/admin/clickhouse/predefined_querylog_queries.py:416]: Ditto comment.

**Review by onkar — APPROVED** [review]
LGTM, minor nit around the interval.

File: snuba/admin/clickhouse/predefined_querylog_queries.py (hunk 1 above)
Comment (xurui-c) [reply snuba/admin/clickhouse/predefined_querylog_queries.py]: done

File: snuba/admin/clickhouse/predefined_querylog_queries.py (hunk 2 above)
Comment (xurui-c) [reply snuba/admin/clickhouse/predefined_querylog_queries.py]: done

**Pull request merged**